GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -688,6 +690,12 @@ public Process GetGitProcess(string command, string workingDirectory, string dot
 
             Process executingProcess = new Process();
             executingProcess.StartInfo = processInfo;
+
+            if (this.ForBackground)
+            {
+                executingProcess.PriorityClass = ProcessPriorityClass.BelowNormal;

```

Comment:
I think that this can only be set after the process has started (and I don't think that's happened yet at this point).

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.priorityclass?view=netframework-4.6.1

>Exceptions
Win32Exception
Process priority information could not be set or retrieved from the associated process resource.
>
>-or-
>
>The process identifier or process handle is zero. (The process has not been started.)

Reply:
And fittingly, the functional tests around the loose objects and packfile maintenance steps failed. Thanks!